### DISCUSSION TITLE
WSA v2303.40000.5.0 has been released

### DISCUSSION THREAD

**Discussion by Almighty188** [LSPosed/MagiskOnWSALocal#506] [General]
Current version(s) of WSA:
2303.40000.5.0 Fast
2303.40000.4.0 Slow/RP(Release Preview)
2302.40000.9.0 Retail

**Comment by CarlGao4** [comment]
Where did you get the version tables?

**Reply by Almighty188** [reply]
I am on Windows Eleven Forums and watch the news in https://www.elevenforum.com/news/ for Windows Subsystem for Android™ news which is based on the news from the top at https://github.com/microsoft/WSA/discussions

It will mention there is a newer version such as now which is already at version 2304.40000.3.0 but the issue is that the version has to be on Microsoft Store and some versions like 2304.40000.3.0 so far is only released to the Windows Subsystem for Android™ Preview Program. 

To see what the current version(s) available on Microsoft Store,
1) Open https://store.rg-adguard.net/ in browser
2) In the pull down menu, either  select "ProductId" and put 9P3395VX91NR in the search box 
*OR*
select “URL (link)” and put https://www.microsoft.com/en-us/p/windows-subsystem-for-android/9p3395vx91nr in the search box
3) Next, select the channel from the second drop-down menu and then click on the check mark
4) Look for the file named MicrosoftCorporationII.WindowsSubsystemForAndroid_*_neutral_~_8wekyb3d8bbwe.msixbundle where * is the version number  
2304 is not available yet since this is going to show what's on the Microsoft Store.  

Repeat step #3 for the different channels as fast should always be the newest build available and then check if slow and RP are the same, the other option should not be as new.  So basically when I see newer versions on adguard.net that is newer than MagiskOnWSALocal last build, I will copy and paste the version numbers into a manually generated text in notepad and then paste it here as the version tables are not automatic and manually generated.

**Reply by CarlGao4** [reply]
That's quite a complicated process...
I thought about doing the same thing, but I believed that you have some simpler methods. Thanks for providing this doable idea.

**Reply by Almighty188** [reply]
It's not that complicated except I wonder how they decide on whether to provide the newer version or not since other than retail, those are all preview channels.

**Reply by CarlGao4** [reply]
Sorry for my poor English, but I can't understand what you mean. Maybe you can explain it in more formal English? 
By the way, now all channels are the same latest version

**Reply by fromhgbwithluv** [reply]
> Sorry for my poor English, but I can't understand what you mean. Maybe you can explain it in more formal English? By the way, now all channels are the same latest version

If you want to get version, you may need user_code for check the version from WSA Insider Program

**Reply by Almighty188** [reply]
> Sorry for my poor English, but I can't understand what you mean. Maybe you can explain it in more formal English? By the way, now all channels are the same latest version

No it's not as the latest you can use is still 2303.40000.5.0 since I just checked.  People who use the non-rooted version are getting 2304.40000.3.0 which is newer:
![image](https://github.com/LSPosed/MagiskOnWSALocal/assets/78993393/6b7e6985-81da-4097-946a-623183cb93d5)
